apis/v1alpha1/cache_cluster.go

```diff
@@ -163,8 +163,6 @@ type CacheClusterSpec struct {
 	// Memcached engine version 1.6.6 on all instances built on the Nitro system
 	// (http://aws.amazon.com/ec2/nitro/).
 	IPDiscovery *string `json:"ipDiscovery,omitempty"`
-	// Specifies the destination, format and type of the logs.
-	LogDeliveryConfigurations []*LogDeliveryConfigurationRequest `json:"logDeliveryConfigurations,omitempty"`
```

Comment:
Removing fields from a CRD is a breaking change that requires a version bump. Let's leave this one for now.

Reply:
To avoid having a field that does not function, I'll add validation to disallow setting this field.